src/pages/removePlantPage/RemovePlant.tsx

```diff
@@ -0,0 +1,44 @@
+import PageComponent from "../../components/pageComponent/PageComponent.tsx";
+import NavItem from "../../components/navItem/NavItem.tsx";
+import WarningText from "../../components/text/WarningText.tsx";
+import CallToActionAsLink from "../../components/callToActionButton/CallToActionAsLink.tsx";
+import { useNavigate, useParams } from "react-router-dom";
+import { pb, PLANTS_COLLECTION } from "../../Backend.constants.ts";
+import { toast } from "react-toastify";
+import CallToActionButton from "../../components/callToActionButton/CallToActionButton.tsx";
+import { useMutation } from "@tanstack/react-query";
+
+const RemovePlant = () => {
+  const { plantId } = useParams();
+  const navigateTo = useNavigate();
+
+  const removePlant = (id) => {
+    pb.collection(PLANTS_COLLECTION)
+      .delete(id)
+      .then(() => {
+        toast.success("Roślinka usunięta z kolekcji!");
```

Comment:
side effects can be nicely managed with react-query

You can use `onSuccess`/`onError` callbacks to manage them

Reply:
in case of error I'm adding text: "Nastąpił błąd podczas usuwania roślinki. Spróbuj proszę jeszcze raz."